src/components/activity-groups-list/activity-group-item/activity-group-amount-change/activity-group-amount-change.tsx

```diff
@@ -62,10 +66,7 @@ export const ActivityGroupAmountChange: FC<Props> = ({ group }) => {
       }
     }
 
-    const positiveDollarSum = roundFiat(new BigNumber(positiveAmountSum));
-    const negativeDollarSum = roundFiat(new BigNumber(negativeAmountSum));
-
-    return { amounts, dollarSums: [positiveDollarSum, negativeDollarSum].filter(sum => !sum.eq(0)) };
+    return { amounts, dollarSums: [positiveAmountSum, negativeAmountSum].filter(sum => sum !== 0) };
```

Comment:
`dollarSums` is not used.  I'm I right?

Reply:
You're wrong https://github.com/madfish-solutions/templewallet-mobile/blob/461c46734c3b9671af4e9b3e1f275662d5720ff8/src/components/activity-groups-list/activity-group-item/activity-group-amount-change/activity-group-amount-change.tsx#L81